spotify/PKGBUILD

```diff
@@ -61,30 +63,28 @@ prepare() {
 }
 
 package() {
-    cd "${srcdir}"
-
     tar -xzf data.tar.gz --no-same-owner -C "${pkgdir}"
 
     # Enable spotify to open URLs from the webapp
-    sed -i 's/^Exec=.*/Exec=spotify --uri=%U/' "${pkgdir}"/usr/share/spotify/spotify.desktop
+    sed -i 's/^Exec=.*/Exec=spotify --uri=%U/' "${pkgdir}/usr/share/spotify/spotify.desktop"
 
-    install -Dm644 "${pkgdir}"/usr/share/spotify/spotify.desktop "${pkgdir}"/usr/share/applications/spotify.desktop
-    install -Dm644 "${pkgdir}"/usr/share/spotify/icons/spotify-linux-512.png "${pkgdir}"/usr/share/pixmaps/spotify-client.png
+    install -Dm 644 "${pkgdir}/usr/share/spotify/spotify.desktop" "${pkgdir}/usr/share/applications/spotify.desktop"
+    install -Dm 644 "${pkgdir}/usr/share/spotify/icons/spotify-linux-512.png" "${pkgdir}/usr/share/pixmaps/spotify-client.png"
 
     for size in 22 24 32 48 64 128 256 512; do
-        install -Dm644 "${pkgdir}/usr/share/spotify/icons/spotify-linux-$size.png" \
+        install -Dm 644 "${pkgdir}/usr/share/spotify/icons/spotify-linux-${size}.png" \
             "${pkgdir}/usr/share/icons/hicolor/${size}x${size}/apps/spotify.png"
     done
 
     # Move spotify binary to its proper location
-    mkdir -p "${pkgdir}"/opt/spotify
+    mkdir -p "${pkgdir}/opt/spotify"
     mv "${pkgdir}/usr/share/spotify" "${pkgdir}/opt/"
 
-    # Symlink spotify binary which is located in /opt
-    ln -sf /opt/spotify/spotify "${pkgdir}/usr/bin/spotify"
+    # Copy custom binary which allows the use of custom flags
```

Comment:
I would rather call this a launch script than a custom binary

Reply:
You're right, more accurate and less scary I guess :stuck_out_tongue: 
Comment edited in https://github.com/christian-heusel/aur/pull/4/commits/184fdfba56fa9d1a3a2dcd4944b428397ac8d005